lib/livebook/secrets.ex

```diff
@@ -1,50 +1,40 @@
 defmodule Livebook.Secrets do
   @moduledoc false
 
+  alias Livebook.Hubs.Provider
   alias Livebook.Storage
   alias Livebook.Secrets.Secret
 
-  @secret_startup_key :livebook_startup_secrets
-
   @doc """
   Get the secrets list from storage.
   """
-  @spec get_secrets() :: list(Secret.t())
-  def get_secrets do
-    startup_secrets = :persistent_term.get(@secret_startup_key, [])
-    storage_secrets = for fields <- Storage.all(:secrets), do: to_struct(fields)
-
-    Enum.concat(storage_secrets, startup_secrets)
+  @spec get_secrets(Provider.t()) :: list(Secret.t())
+  def get_secrets(hub) do
+    for fields <- Storage.all(hub.id),
```

Comment:
I think it should be `Storage.all(:hub_secrets, hub_id: hub.id)` but I don't remember if we can do filters like this. We can always filter it after the fact anyway.

Reply:
We store each attribute separately, so to filter this way we need to fetch everything either way. In this case it should be fine since it's pretty much always personal hub?

Reply:
We may store other hubs in the future but the hub_id will address that. In all cases, it will be super fast to just read all from memory anyway, so `:hub_secrets` is good enough for me. :+1: